### PR TITLE
Deprecated UIColoredView + SwiftLint warnings on master

### LINKED ISSUE #270: UIColoredView
Today we have an `UIColoredView.swift`, that returns a 100x100 `UIView` with a random `UIColor`.

@goktugyil created this and I'd like to question if it is really necessary for **EZSwiftExtensions** since I don't see many use cases for this, especially because we already have extensions that already make it easy to do it:

``` swift
let view = UIView(x:0,y:0,w:100,h:100)
view.backgroundColor = UIColor.randomColor()
```

### PR DESCRIPTION
**Pull request by piv199**
Fixes #270

### CODE REVIEW

**Comment by lfarah** [comment]
> The command "bundle exec danger --fail-on-errors=true" exited with 1.

File: Sources/ArrayExtensions.swift [hunk 1]
```diff
@@ -19,9 +19,9 @@ public func ==<T: Equatable>(lhs: [T]?, rhs: [T]?) -> Bool {
 }
 
 extension Array {
-    
+
     ///EZSE: Get a sub array from range of index
-    public func get(at range:ClosedRange<Int>) -> Array {
+    public func get(at range: ClosedRange<Int>) -> Array {
```
Comment (lfarah) [inline Sources/ArrayExtensions.swift]: I think you should remove changes in this file

File: Sources/CGFloatExtensions.swift [hunk 2]
```diff
@@ -47,17 +47,17 @@ extension CGFloat {
     public static func radiansToDegrees(_ angleInDegrees: CGFloat) -> CGFloat {
         return (180.0 * angleInDegrees) / .pi
     }
-    
+
```
Comment (lfarah) [inline Sources/CGFloatExtensions.swift]: I think you should remove changes in this file

Comment (piv199) [inline Sources/CGFloatExtensions.swift]: remove this

File: Sources/ArrayExtensions.swift (hunk 1 above)
Comment (piv199) [inline Sources/ArrayExtensions.swift]: remove this

File: Sources/CGFloatExtensions.swift [hunk 3]
```diff
@@ -47,17 +47,17 @@ extension CGFloat {
...
     /// EZSE: Returns a random floating point number between 0.0 and 1.0, inclusive.
     public static func random() -> CGFloat {
         return CGFloat(Float(arc4random()) / 0xFFFFFFFF)
     }
-        
+
```
Comment (piv199) [inline Sources/CGFloatExtensions.swift]: remove this

File: Sources/CGFloatExtensions.swift [hunk 4]
```diff
@@ -47,17 +47,17 @@ extension CGFloat {
...
     /// EZSE: Returns a random floating point number in the range min...max, inclusive.
     public static func random(within: Range<CGFloat>) -> CGFloat {
         return CGFloat.random() * (within.upperBound - within.lowerBound) + within.lowerBound
     }
-    
+
```
Comment (piv199) [inline Sources/CGFloatExtensions.swift]: remove this

**Comment by lfarah** [comment]
@piv199 I'm so dumb.. You made the spacing changes **because** of Swiftlint. Sorry about that!

**Comment by piv199** [comment]
@lfarah it's still should be separate, nevertheless

**Comment by lfarah** [comment]
@piv199 can we just deprecate ```UIColoredView```? The Swiftlint warnings will be fixed in #306

**Comment by piv199** [comment]
@lfarah I will be on the weekend, the fast answer is yes

**Pull request merged**